Core/include/Acts/Utilities/Identity.hpp

```diff
@@ -0,0 +1,22 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <utility>
+
+namespace Acts {
+
+/// @brief Function object which maps a value to itself by perfect forwarding
+/// TODO Replace Identity with std::identity on C++20
```

Comment:
If `std::identity` implements the same interface, we could make this with a pragma already and alias of std::identity,  I always fear that we'll forget to remove this sort of code.